codebuild/browser.yml

```diff
@@ -6,11 +6,12 @@ env:
 
 phases:
   install:
-    runtime-versions:
-      nodejs: latest
     commands:
+      - n 18
       - npm ci --unsafe-perm
       - npm run build
   build:
     commands:
+      - npm -v
+      - node -v
```

Comment:
i assume these are for debugging purposes?

Reply:
Yes, I left them in since it is using `n` to manually update the version.